scripts/build_release_archive.sh

```diff
@@ -37,7 +37,18 @@ CLONE_DIR=`mktemp -d`
 
 # Download the repository
 cd "${CLONE_DIR}"
-git clone -b "${1}" --depth 1 "https://github.com/google/${REPO_NAME}.git" || exit
+git clone -b "${1}" "https://github.com/tst-eclamar/${REPO_NAME}.git" || exit # TEMPORARY: tst-eclamar => google
+
+cd "${CLONE_DIR}"/earthenterprise/earth_enterprise/src
+git remote add upstream git://github.com/google/earthenterprise.git # TEMPORARY
+git fetch upstream # TEMPORARY
+git pull upstream master # TEMPORARY
+# generate version files based on tags within git
+scons version_files
+cd ../..
```

Comment:
From a code safety perspective using relative paths in cd commands inside of scripts can be dangerous, especially if the script uses default settings and does not stop executing when an error is encountered. Basically, you don't really know that you're in the correct directory before executing a relative path. Also, if you get into the habit of using cd, you may get into the worse habit of following it up with a command that uses wildcards, such as `rm *`. These are just some things to be mindful of.

I suggest getting into the habit of using the style above `cd "${CLONE_DIR}"/earthenterprise` rather than `cd ../..`. Also, I recommend enabling stop execute on error by adding the `set -e` command after the file header comment.